signal_test.go

```diff
@@ -0,0 +1,490 @@
+package dbus_test
+
+import (
+	"errors"
```

Comment:
Break apart imports into three groups:

```
import (
    stdlib packages

    third-party packages

    local packages
)
```

So in this case:

```
import (
    "errors"
    "testing"
    "time"

    "github.com/godbus/dbus/introspect"
)
```

I didn't carry over the `github.com/godbus/dbus` import since this file should actually be a part of that package (see comment above about package naming).